### PR TITLE
Removing like most of the image commands that nobody even uses anymore

### PR DESCRIPTION
**Pull request by LoliconSlayer**
If you object to a command being removed please provide a reason other than "I like it" or any variations of such before my eyes glaze over due to the sandpaper your words apply to my brain.

### CODE REVIEW

**Comment by Hopekz** [comment]
Reason for not deleting them:
There's no downside of keeping these around. 

It doesn't even take up space on your PC because yogbot isn't stored on your PC.

Really seems like you just woke up one day and chose to erase years worth of inside jokes/history.

By history I mean people that have greatly contributed to this project are sort of "remembered" through these. Almost like a mini legacy in our server

**Comment by wejengin2** [comment]
You're removing yogs history, some of these people are still around too.

Also i like corgis

**Comment by JamieD1** [comment]
The hell

**Comment by nichlas0010** [comment]
lmao

File: models/Discord/Commands/DiscordCommandAsh.js [hunk 1]
```diff
@@ -1,31 +0,0 @@
-const DiscordCommand = require('../DiscordCommand.js');
-var Discord = require('discord.js');
-
-class DiscordCommandAsh extends DiscordCommand {
```
Comment (AshCorr) [inline models/Discord/Commands/DiscordCommandAsh.js:4]: I cant believe you've done this

**Pull request closed without merging**